app/build.gradle.kts

```diff
@@ -12,8 +10,8 @@ plugins {
 }
 
 if (gradle.startParameter.taskRequests.toString().contains("Standard")) {
-    apply(mapOf("plugin" to "com.google.gms.google-services"))
-    apply(mapOf("plugin" to "com.google.firebase.crashlytics"))
+    // apply(mapOf("plugin" to "com.google.gms.google-services"))
+    // apply(mapOf("plugin" to "com.google.firebase.crashlytics"))
```

Comment:
why is this commented out?

Reply:
Disregard that, it was a local-only change, I reverted